src/get_clades_to_model.py

```diff
@@ -135,17 +138,36 @@ def get_clades(
     # sort clade list before returning
     variants.sort()
 
-    return variants
+    return variants, prop_dat
 
 
-def get_metadata(ct: CladeTime, current_time: str) -> dict[str, dict | str]:
+def get_metadata(ct: CladeTime, sequence_counts: pl.LazyFrame) -> dict[str, dict | str]:
     """Create metadata to store with the clade list."""
+    current_time = ct.sequence_as_of.isoformat(timespec="seconds")
     metadata: dict[str, dict | str] = dict(created_at=current_time)
 
+    # add metadata about nextstrain pipeline used to generate sequence metadata
+    # that informs our list of clades to model
     ncov_metadata = ct.ncov_metadata
     ncov_metadata["metadata_version_url"] = ct.url_ncov_metadata
     metadata["ncov"] = ncov_metadata
 
+    # add metadata about the number of sequences used to create the
+    # list of modeled clades
+    sequence_metadata: dict[str, dict | int] = {}
+    total_sequences = sequence_counts.select("count").sum().collect().item()
+    sequences_by_clade = (
+        sequence_counts.select("clade", "count")
+        .group_by("clade")
+        .agg(pl.col("count").sum())
+        .sort("clade")
+        .collect()
+    )
+
+    sequence_metadata["total_sequences"] = total_sequences
```

Comment:
Could we change this field to be something more specific and descriptive like "total_sequences_last_3_weeks" or something like that?